### ISSUE TITLE
Julia Version Compatibility

### ISSUE THREAD

**Issue by mehalter** [sisl/BayesNets.jl#84]
The current `Project.toml` has an error that says that it supports `julia >= 0.7`. This is incorrect. Could you please update it in different versions to reflect to actually supported versions? Version 3.0.0 for example works on Julia v1.0, but 3.1.0 only works on Julia 1.1.

This compatibility error in the Project.toml causes the registry for Julia 1.0 to pull the wrong version and BayesNets doesn't actually work anymore on Julia 1.0.

**Comment by mossr** [comment]
@tawheeler is this as simple as updating the Project.toml to require Julia >= 1.1.0 or should we tag a new BayesNets.jl version in the process?

**Comment by mykelk** [comment]
Yeah, you'll need to update it along with increment the version number in a commit. Then you will want to add a comment like [this](https://github.com/sisl/BayesNets.jl/commit/1f4426077dcd23bf905c83356ea8330d5678644b) to trigger JuliaRegistrator. Then the JuliaTagBot will automatically tag a release once it is merged into the registry.

**Comment by mossr** [comment]
@mykelk want me to submit a pull request for this, or continue on master?

**Comment by mossr** [comment]
@mykelk disregard that, I'm going down the pull request route.

**Comment by mykelk** [comment]
@mossr either way is fine with me.

**Comment by mehalter** [comment]
I would recommend you also tag a version that supports 1.0 and had the correct compatibility line so that the package has a version that supports the LTS version of Julia.

**Comment by mossr** [comment]
@mehalter BayesNets v3.1.0 works on Julia v1.0 for me. You mentioned v3.1.0 only works on Julia v1.1—what were you seeing that suggested that?
